pophory-iOS/Global/Components/Base/BaseInputView.swift

```diff
@@ -22,12 +22,12 @@ class BaseSignUpView: UIView{
     
     lazy var headerLabel: UILabel = {
         let label = UILabel()
-        label.text = "만나서 반가워\n너의 이름이 궁금해!"
+        label.text = "시작할 준비가 되었다면\n너의 이름을 알려줘!"
         label.textColor = .black
-        label.font = .h1
+        label.font = .head1Medium
         label.numberOfLines = 0
         label.setTextWithLineHeight(lineHeight: 34)
-        label.asColor(targetString: "너의 이름", color: .pophoryPurple)
+        label.applyColorAndBoldText(targetString: "너의 이름", color: .pophoryPurple, font: .head1Medium, boldFont: .head1Bold)
```

Comment:
난 이걸 이렇게 써 (근데 이 방식이 localizable 적용할 땐 정말 안 좋긴 해)
나중에 attributedString 쓰는 거 다 맞추면 좋겠다!!

```suggestion
        label.numberOfLines = 0
        label.setTextWithLineHeight(lineHeight: 34)
        label.attributedText = NSMutableAttributedString()
            .regular("시작할 준비가 되었다면\n", font: .head1Medium, color: .pophoryBlack)
            .regular("너의 이름", font: .head1Bold, color: .pophoryPurple)
            .regular("을 알려줘!", font: .head1Medium, color: .pophoryBlack)
```

Reply:
오.. 확실히 깔끔하네요! 혹시 localizable이 다국어지원 처리하는거 맞아???

Reply:
맞습니당